onnxruntime/test/onnx/main.cc

```diff
@@ -324,6 +364,11 @@ int real_main(int argc, char* argv[], Ort::Env& env) {
             return -1;
           }
           break;
+        case 'C':
+          if (!ParseSessionConfigs(ToUTF8String(optarg), session_config_entries)) {
+            return -1;
+          }
+          break;
```

Comment:
Is usage of this documented anywhere?

Reply:
it is in the helper message ( --help ) at the top of this file.

and this change makes it consistent with onnxruntime_perf_test